spec/rails_helper.rb

```diff
@@ -35,6 +35,18 @@
   # instead of true.
   config.use_transactional_fixtures = true
 
+  # database_cleaner config
+    config.before(:suite) do
+    DatabaseCleaner.strategy = :transaction
+    DatabaseCleaner.clean_with(:truncation)
+  end
+
+  config.around(:each) do |example|
+    DatabaseCleaner.cleaning do
+      example.run
+    end
+  end
```

Comment:
Thanks for putting this in! But we don't actually need the `around(:each)` one. You can delete lines 44-48. Be sure line 39 is indented properly - should be one over to the left